components/ExpenseListDetail/index.js

```diff
@@ -1,25 +1,22 @@
 import React from "react";
-import styled from "styled-components";
+import { Icon } from "@iconify/react";
 
-const ExpenseDetailContainer = styled.div`
-  display: flex;
-  width: 100%;
-`;
+import {
+  ExpenseDetailContainer,
+  ExpenseDetailItem,
+  CustomLink,
+} from "./ExpenseListDetail.styled";
 
-const ExpenseDetailItem = styled.div`
-  flex: 1;
-  padding: 8px;
-  text-align: left;
-  display: flex;
-  align-items: center;
-`;
-
-function ExpenseListDetail({ name, description, amount }) {
+function ExpenseListDetail({ id, name, description, amount }) {
   return (
     <ExpenseDetailContainer>
       <ExpenseDetailItem>{name}</ExpenseDetailItem>
       <ExpenseDetailItem>{description}</ExpenseDetailItem>
       <ExpenseDetailItem>{amount} €</ExpenseDetailItem>
+
+      <CustomLink href={`./form/${id}`}>
```

Comment:
it is better not to work with relative links. Better to do it like this:

```suggestion
      <CustomLink href={`form/${id}`}>
```